src/routes/blog/posts/announcing-files-v2-5-20/+page.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Announcing Files, version 2.5.20
+description: Push Git commits to remote repositories, splash screen, updated icons, support for ownCloud.
+thumbnail: /blog-resources/files2-5-20/HeroImage.jpg
+date: 7/24/2023
+author: files-community
+---
+
+We're excited to announce that version 2.5.20 is now available. If you already have Files, you'll get a notification in the app to update. Otherwise, you can download Files for free from our [download page](/download/). You can also purchase Files on the [Microsoft Store](ms-windows-store://pdp/?ProductId=9nghp3dx8hdx&cid=FilesWebsite) as a way to support for project.
+
+**TL;DR:** This update adds support for pushing Git commits, an improved startup experience, updated toolbar and status icons, support for editing tags from the Details pane, and more. Continue reading to learn more about these changes.
+
+## What's new in v2.5.20
+
+### Push Git commits to remote repositories
+
+To further improve our integration with Git, we're excited to be adding support for pushing commits to remote repos. The Status Bar will display you how many local and remote commits you have, and you can click on it to access the sync, pull, and push options. We hope this feature makes it easier for you to collaborate with other developers and keep your code up to date.
+
+<figure>
+    <img src="/blog-resources/files2-5-20/GitPush.png" alt="Push commits to remote repos." />
+</figure>
+
+### Improvements to the startup experience
+
+Startup performance is one of our the highest priorities and something we're working on improving. We'd like the progress to be faster but unfortunately there are some limitations in the WinAppSdk framework. We hope these limitations will be resolved in future versions of WinAppSdk, but in the meantime, we've been working on a lot of smaller optimizations that we hope will add up over time. We're also adding a splash screen when the app is opened to indicate that the app is loading.  
```

Comment:
"Startup performance is one of our the highest" should be "Startup performance is one of our highest"